.github/workflows/on-push.yml

```diff
@@ -0,0 +1,24 @@
+name: push-event
+
+on:
+  push:
+
+jobs:
+  job:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out github-actions repo
+        uses: actions/checkout@v4
+        with:
+          repository: aerius/github-actions
+          path: aerius-github-actions
+          ref: v1.1
+
+      - uses: aerius/github-actions/events/push-event-action@v1.1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+          NEXUS_USERNAME: ${{ secrets.NEXUS_USERNAME }}
+          NEXUS_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+          ROOT_POM_DIRECTORY: repo
```

Comment:
```suggestion
```
This entry can go.. It's the default setting.. We only set it when the root pom isn't the root of the repository.